### DISCUSSION TITLE
能否屏蔽“新版直播间”开启弹窗

### DISCUSSION THREAD

**Discussion by Lssg97** [the1812/Bilibili-Evolved#2237] [功能建议 / Ideas]
每次进入 直播中心-我的直播间 后，都会对未开启新版直播间的UP主弹出如下弹窗，一遍又一遍不胜其烦。请问有技术能力屏蔽掉它吗？
![image](https://user-images.githubusercontent.com/47817993/131450694-b4bf31cf-a9c7-4bc4-94e0-da6adbe6ba34.png)

**Comment by the1812** [comment]
恐怕来不及了...
![image](https://user-images.githubusercontent.com/26504152/133247503-f86780f7-62fc-473d-8a10-42dbb4e6e08f.png)

**Reply by Lssg97** [reply]
噫~ 好的（捂脸）
